.azure/validate.azure-pipelines.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Validate the code'
+
+pr:
```

Comment:
I do no think it is necessary to trigger the pipeline always. Especially we are on a free plan. Do you think trigger it on PR to master and develop enough?

Reply:
Yes, that should be enough for sure. I don't even know when else we could run it :)